### ISSUE TITLE
[Bug] [Demo App] Missing Accessibility Label for Image

### ISSUE THREAD

**Issue by Tayebsed93** [Orange-OpenSource/ouds-ios#212]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 Pro Max

### OS Version

iOS 18

### App version

v0.4.0

### App orientation

Portrait

### Device language

English

### App language

English

### Describe the issue

SwiftLint has flagged an accessibility issue: images that provide context should have an accessibility label or be explicitly hidden from accessibility. The violation occurs under the rule `accessibility_label_for_image`.

![Image](https://github.com/user-attachments/assets/c5fdc713-e62a-49fd-b86e-12f923902ceb)

Please review the code and ensure that all images with contextual significance include an appropriate accessibility label, or mark them as hidden for accessibility when necessary.

SwiftLint message:
`Accessibility Label for Image Violation: Images that provide context should have an accessibility label or should be explicitly hidden from accessibility (accessibility_label_for_image)`

### Expected Behavior

This issue affects `VoiceOver` users in both the `ThemeSelection.swift` and `ShowcaseElementsPage.swift` files. Images without accessibility labels are not properly announced by screen readers. Adding appropriate labels ensures that VoiceOver can describe the images, providing necessary context for visually impaired users and enhancing the app’s accessibility.

**Example** :
```
Image(systemName: "paintpalette")
    .accessibilityLabel("app_common_theme_image_a11y")
```

This ensures that the image is properly announced by screen readers, enhancing accessibility compliance and providing necessary context for visually impaired users.

### Accessibility settings

This issue affects `VoiceOver` users, as images without accessibility labels will not be properly announced by screen readers. Providing an accessibility label ensures that VoiceOver can describe the image, giving context to visually impaired users and improving the overall accessibility of the app.
